release-notes/6.0/api-diff/.Net/6.0.0_System.IO.Compression.md

```diff
@@ -0,0 +1,74 @@
+# System.IO.Compression
+
+``` diff
+ namespace System.IO.Compression {
+     public sealed class BrotliStream : Stream {
++        public override int ReadByte();
++        public override void WriteByte(byte value);
+     }
+     public enum CompressionLevel {
++        SmallestSize = 3,
+     }
+     public class DeflateStream : Stream {
+-        public override IAsyncResult BeginWrite(byte[] array, int offset, int count, AsyncCallback asyncCallback, object asyncState);
++        public override IAsyncResult BeginWrite(byte[] buffer, int offset, int count, AsyncCallback asyncCallback, object asyncState);
```

Comment:
since the array parameter was renamed to buffer (and the only changed publicly visible change) here cant we just exclude them from the diff?

Reply:
we generally keep the name changes as its breaking changed for anyone using the named arguments.